### ISSUE TITLE
allow user options for systemd timers

### ISSUE THREAD

**Issue by jgilbert2017** [kdave/btrfsmaintenance#104]
please add support for a system specific /etc/ config file to allow additional options to be set inside the systemd timer script by the command btrfsmaintenance-refresh-cron.sh timer.

my use case is that i want to set RandomizedDelaySec= but i would like to do so without editing btrfsmaintenance-refresh-cron.sh so that i dont lose changes if the script is ever upgraded.

**Comment by DaAwesomeP** [comment]
For Systemd you can use overrides: https://www.freedesktop.org/software/systemd/man/systemd.unit.html#id-1.14.3

**Comment by kdave** [comment]
Yes, the systemd overrides should be used for tuning the services or timers.

**Comment by kdave** [comment]
This should be better documented too.
